phoenix-core/src/main/java/org/apache/phoenix/schema/types/FileSystemBasedLobStore.java

```diff
@@ -33,6 +33,12 @@
 
     private static final String ROOT_DIRECTORY = "/tmp/";
 
+    //todo consider not preinitiializing
```

Comment:
+1 to that. We can skip worrying about thread-safety for the hackathon, but can definitely have INSTANCE == null check and only initialize if required